.github/ISSUE_TEMPLATE/ticket_template1.md

```diff
@@ -0,0 +1,27 @@
+🚨Please review the [guidelines for ticket creation](../HOWTOWRITEATICKETS.md) if needed
```

Comment:
Better to use [contributing guidelines](https://blog.github.com/2012-09-17-contributing-guidelines/).

Reply:
Thanks for the suggestion. I will add this to the template.